### DISCUSSION TITLE
EthernetWebServer + Modbus TCP

### DISCUSSION THREAD

**Discussion by dirkdickler** [khoih-prog/EthernetWebServer#58] [General]
I would like ask,
When I use EthernetWebServer 2,2,0 on W5500  there is 8
socket  and I need  ONE socket for Modbus TCP on port 502 , is any way to
coexistence your EthernetWebServer and  socket ModbusTCP on port 502.
- EthernetWebServer  will use 7 socket
- Modbus TCP will use one socket
What should I do for this ?Thank you for your job.

**Comment by khoih-prog** [comment]
It's totally possible, but you have to write your code accordingly to merge the `ModbusTCP` with [**Ethernet_Generic** library](https://github.com/khoih-prog/Ethernet_Generic) because the limitation is set by W5500.

You can post the question to ask for more help in Arduino Forum, etc. as I don't have time to spend on this. Hopefully someone will help you in this **Discussion**.

Good Luck,
